core/downloadfromdcthread.cpp

```diff
@@ -49,21 +50,38 @@ void DownloadThread::run()
 	qDebug() << "Finishing the thread" << errorText << "dives downloaded" << downloadTable.nr;
 }
 
+static void fill_supported_mobile_list()
+{
+	mobileProductList["Heinrichs Weikamp"] =
+			QStringList({{"OSTC Sport"}, {"OSTC 2N"}, {"OSTC 3"},
+				     {"OSTC 3+"}, {"OSTC 4"}});
```

Comment:
On Android, all HW computers are potentially supported (they are all FTDI, BT, or BLE).
On iOS only the latest BLE ones are supported (and I need to go back and re-read their emails, just like Shearwater they may have computers that added BLE half way through the production run